docs/hpc/pilot/cx3-phase2.md

```diff
@@ -127,6 +127,15 @@ If you are not using the base environment, we recommending you disable automatic
 ```
 
 Now whenever you load conda using the eval command only the conda commands will be loaded, Python won't change until you "source activate" the environment you need for your workflow. 
+
+## Applications
+### JupyterHub
+CX3 Phase 2 runs an updated and improved Jupyter service that better manages resource usage. Now, both memory and CPU limits are controlled and restricted to the amount of resource that you request when you start your job. Should you go over these limits, you will see a message that your kernel has been killed and that it will restart.
```

Comment:
```suggestion
CX3 Phase 2 runs an updated and improved Jupyter service that better manages resource usage. Both memory and CPU limits are controlled and restricted to the amount of resource that you request when you start your job. Should you go over these limits, you will see a message that your kernel has been killed and that it will restart.
```

Reply:
Not sure if Now is needed or not.